gradlew

```diff
@@ -32,10 +32,10 @@
 #       Busybox and similar reduced shells will NOT work, because this script
 #       requires all of these POSIX shell features:
 #         * functions;
-#         * expansions �$var�, �${var}�, �${var:-default}�, �${var+SET}�,
-#           �${var#prefix}�, �${var%suffix}�, and �$( cmd )�;
-#         * compound commands having a testable exit status, especially �case�;
-#         * various built-in commands including �command�, �set�, and �ulimit�.
+#         * expansions «$var», «${var}», «${var:-default}», «${var+SET}»,
+#           «${var#prefix}», «${var%suffix}», and «$( cmd )»;
+#         * compound commands having a testable exit status, especially «case»;
+#         * various built-in commands including «command», «set», and «ulimit».
```

Comment:
nit: I'm having a hard time telling a single difference in these lines and line 4 above. Did something significant change in this file? Otherwise I'd suggest leaving it out of the PR (e.g. by doing a `git checkout HEAD~1 gradlew` + an amend)

Reply:
After receiving an email with these lines quoted, I notice it probably changed the file encoding, so this change should be fine